strings/boomer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "boomer": [
+	"@pick(kids) these days have it too easy!",
+	"Do I look like I know what a @pick(file) is!?",
+	"How do I open a @pick(file) again?",
+	"Unlike you snowflakes, I'm not offended so easily.",
+	"Back in my day...",
+	"When I was your age...",
+	"This generation can't take a joke.",
+	"Why are @pick(kids) these days so @pick(sad) all the time?",
+	"These damn @pick(kids) need to get a @pick(impossible) for once!",
+	"It's simply a matter of showing up, looking the manager in the eye, giving him a firm handshake and telling him you want the job."
+    ],
```

Comment:
"You'll never get anywhere in life without a degree."
"Healthcare isn't really _that_ expensive."

Reply:
yes